librz/util/bitvector.c

```diff
@@ -209,26 +213,62 @@ RZ_API ut32 rz_bv_copy(RZ_NONNULL const RzBitVector *src, RZ_NONNULL RzBitVector
  * \param nbit ut32, control the size of copy (in bits)
  * \return copied_size ut32, Actual copied size
  */
+
 RZ_API ut32 rz_bv_copy_nbits(RZ_NONNULL const RzBitVector *src, ut32 src_start_pos, RZ_NONNULL RzBitVector *dst, ut32 dst_start_pos, ut32 nbit) {
-	rz_return_val_if_fail(src && dst, 0);
+    rz_return_val_if_fail(src && dst, 0);
 
-	ut32 max_nbit = RZ_MIN((src->len - src_start_pos),
-		(dst->len - dst_start_pos));
+    // Determine the chunk size (word size) dynamically
+    const ut32 chunk_size = SIZE_OF_UNSIGNED_LONG * CHAR_BIT; // Word size in bits
```

Comment:
You can move this whole expression into a `#define`. Macros allow some basic math

```
#define RZ_BV_CHUNK_SIZE (sizeof(unsigned long) * CHAR_BIT)
```